gamebook/creature.py

```diff
@@ -0,0 +1,13 @@
+class Creature(object):
+    def __init__(self, traits):
+        self.traits = traits
+        self.is_alive = True
```

Comment:
Why not use health? (then if health <= 0)

Reply:
i thought it is more "user friendly" this way. like this the user/coder to passes less arguments when building a creature instance.